ibis/backends/druid/registry.py

```diff
@@ -14,6 +14,25 @@
 
 operation_registry.update(sqlalchemy_window_functions_registry)
 
+
+def _sign(t, op):
+    arg = op.arg
+    cond1 = ops.Where(ops.Greater(arg, 0), 1, -1)
+    cond2 = ops.Where(ops.Equals(arg, 0), 0, cond1)
+    return t.translate(cond2)
+
+
+def _join(t, op):
+    sep = t.translate(op.sep)
+    values = list(map(t.translate, op.arg))
+    args = []
+    for i, value in enumerate(values):
+        args.append(value)
+        if i != len(values) - 1:
+            args.append(sep)
+    return sa.func.concat(*args)
```

Comment:
```suggestion
    return sa.func.concat(*toolz.interpose(sep, values))
```

You may need to `import toolz` at the top of the file as well.